content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,103 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing ways to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+projects</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+DVC uses the Google Drive API to synchronize your DVC project data with this
+type of remote storage, so it's subject to certain usage limits and quotas,
+which by default are shared with other GDrive remote storage users. For heavy
+use, it's highly recommended to
+[connect using a custom Google Cloud project](#using-a-custom-google-cloud-project),
+which puts you in control of these limits.
+
+Having your own GC project, it's also possible to
+[use a service account](#using-service-accounts) for automating tasks that need
+to establish GDrive remote connections (e.g. CI/CD).
+
+## Quick start
+
+To start using a Google Drive remote, you only need to add it with a
+[valid URL format](#url-format). Then use any DVC command that needs it (e.g.
+`dvc pull`, `dvc fetch`, `dvc push`). For example:
+
+```dvc
+$ dvc remote add -d gdremote gdrive://0AIac4JZqHhKmUk9PDA/dvcstore
+$ dvc push
+
+Go to the following link in your browser:
+
+    https://accounts.google.com/o/oauth2/auth # ... copy this link
+
+Enter verification code: # <- enter resulting code
+```
+
+See [Authorization](#authorization) for more details.
+
+## URL format
+
+There's a few alternatives to construct a GDrive remote URL for different uses,
+such as a folder or subfolder in root, shared folders not owned by your account,
+etc. The URL is formed with a _base_, and an optional _path_ to an **existing**
+folder i.e. `gdrive://<base>/path/to/folder`. The base can be one of:
+
+1. _Folder ID_ - unique identifier for every Google Drive folder, including
+   [shared folders](https://support.google.com/drive/answer/7166529) and
+   [shared drives](https://support.google.com/a/users/answer/9310351) (these two
+   can only be referenced by ID).
+
+   ```dvc
+   $ dvc remote add gdfolder gdrive://0AIac4JZqHhKmUk9PDA
+   ```
+
+   or
+
+   ```dvc
+   $ dvc remote add gdsubfolder \
```

Comment:
I'mt not sure about using these names like gdsubfolder, etc - It might look like it is some convention. Probably it's better to keep it simple everywhere - `myremote`

Reply:
OK, changed.